apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console/GoogleCloudConsoleFormatter.cs

```diff
@@ -177,6 +185,35 @@ private static void WriteKeyValuePairs(Utf8JsonWriter writer, IEnumerable<KeyVal
         writer.WriteEndObject();
     }
 
+    private void MaybeWriteTraceInformation(Utf8JsonWriter writer)
+    {
+        if (string.IsNullOrWhiteSpace(_options.TraceGoogleCloudProjectId))
+        {
+            return;
+        }
+
+        var activity = Activity.Current;
+        if (activity == null)
+        {
+            return;
+        }
+
+        var traceId = activity.TraceId;
+
+        // The `Equals` method of the `ActivityTraceId` struct uses the _hexString field to compare two ActivityTraceId instances.
+        // The ActivityTraceId struct has a default value with an uninitialized _hexString field.
+        // So, `Equals` method doesn't work as expected. String comparision is unfortunate but currently the only available workaround.
```

Comment:
I'm not sure what this means, that Equals does not behave as expected.

Regardless, I'm thinking maybe we should just check IsStopped, and we can do that, alongside the other conditions.

Reply:
Updated the comment with the issue link in `ActivityTraceId`

As discussed, `Activity.IsStopped` is available in .NET 7 and 8. Additionally, I think when IsStopped is called, it sets the `Activity.Current` to its Parent activity which may or may not be null.